modules/Behaviour/behaviour_manage_edit.php

```diff
@@ -232,6 +234,13 @@
                 //Lesson link
                 $lessons = [];
                 $minDate = date('Y-m-d', (strtotime($values['date']) - (24 * 60 * 60 * 30)));
+                $resultSelect = $container->get(CourseGateway::class)->selectLessonByStudentAndDate(date('Y-m-d', strtotime($values['date'])), $minDate, $session->get('gibbonSchoolYearID'), $values['gibbonPersonID']);
```

Comment:
This looks similar to `selectPlannerLessonsByStudent`, and should likely go in the same gateway and use the same method signature + date.